src/addons/transitions/ReactTransitionGroup.js

```diff
@@ -56,17 +55,9 @@ class ReactTransitionGroup extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    var nextChildMapping;
-    if (__DEV__) {
-      nextChildMapping = ReactTransitionChildMapping.getChildMapping(
-        nextProps.children,
-        ReactAddonsDOMDependencies.getReactInstanceMap().get(this)._debugID
-      );
```

Comment:
I opted to just remove this dependency. Its only use is to add the stack trace in this particular case. I think we can live without it like it was in 0.14 (and it wouldn't be able to do this in userland anyway).